apps/search/test/unit/search_test.js

```diff
@@ -83,7 +90,7 @@ suite('search/search', function() {
       Search.init();
       MockNavigatormozApps.mTriggerLastRequestSuccess();
       assert.equal(MockNavigatormozApps.mLastConnectionKeyword,
-        'search-results');
+        'app-metrics');
```

Comment:
I have moved the check for the app-metrics into the separate file for maintaining the app-metrics port as per Dale's comment.